src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -194,18 +201,18 @@ private VCWithNewAlleles(final VariantContext vc, final List<Allele> newAlleles,
             this.isSpanningEvent = isSpanningEvent;
         }
 
-        public List<Allele> filterAllelesForFinalSet() {
+        public Stream<Allele> filterAllelesForFinalSet() {
```

Comment:
So this is a public method on a private inner class.  The scope is really irrelevant because java allows the outer class to reach into the inner class private methods anyway, but I figured it's better to still signal which are internal to the class and which are meant to be accessed by the outer class.  That's why it's public.

I changed it to avoid materializing the list at the end of the call because it seemed like a wasteful allocation.  It's a micro-optimization but since the only use case is in a stream operation in the outer class it seems silly to change it back.

Let me  know what you want me to do.  I think it's fine how it is, but I'm willing to change it if you prefer.